package.json

```diff
@@ -34,5 +36,7 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
-}
+  },
+  "keywords": [],
```

Comment:
@harsha14gupta  Add few keywords like Girl Code It, Women in Tech, community, developer etc.